### DISCUSSION TITLE
creating shared library with prisma

### DISCUSSION THREAD

**Discussion by mohammadjavad948** [prisma/prisma#14452] [Q&A] [answered]
Hi i want to have a shared library that contains all database related stuff (migrations, seeds, schema) and use the library in other projects. how should i do it?

**Comment by arthurfiorette** [comment, marked as answer]
I've done it in the past by changing the generated client output to inside the source folder and exporting it in the index.js file, this way npm would include all generated code. Your only problem is that, by pre generating your whole client, you need to explicitly say what types of [binaries it should support](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
